lib/media/track/index.js

```diff
@@ -58,6 +56,14 @@ class Track extends EventEmitter {
       }
     });
   }
+
+  /**
+   * The {@link Track}'s ID.
+   * @property {Track.ID}
+   */
+  get id() {
+    return this._id;
+  }
```

Comment:
Why are we making this change? Is `_id` mutable now?

Reply:
@markandrus This is because we need to override this getter method in RemoteMediaTrack to print the deprecation warning.